classes/SuperDirt.sc

```diff
@@ -284,14 +284,21 @@ SuperDirt {
 			}
 		};
 
-
 		netResponders.add(
+			OSCFunc({ |msg, time, tidalAddr|
+				var name = msg[1].asSymbol;
+				var synth = SynthDescLib.global.at(name);
+				var controls = synth.controls;
+				var controlNames = msg[1];
+				controls.do{|control| controlNames = controlNames ++ " " ++ control.name};
+				tidalAddr.sendMsg("/dirt/synth-info/reply", controlNames);
```

Comment:
One thing: don't you also want to send the default values over? Or is that not useful on the tidal side ...

Reply:
That might be useful for the human user to know. There could be more info, like the normal range of the parameter, whether it can be controlled via a bus and so on.. So maybe we should think a bit ahead and flesh out https://github.com/musikinformatik/SuperDirt/issues/199 a bit first.

(This PR came from a [question on the forum](https://club.tidalcycles.org/t/synthdef-argument-posting-function/2906?u=yaxu))

Reply:
OK, I'm going over there …